src/main/java/com/rtb/projectmanagementtool/user/UserController.java

```diff
@@ -24,4 +24,16 @@ public UserController(HashSet<UserData> users) {
     }
     return users;
   }
+
+  public void addUser(
+      DatastoreService datastore, long userID,
+      long AuthID, String userName,
```

Comment:
My suggestion, controller shouldn't know too much about specifically what fields UserData owns (ie. what the data model looks like exactly).  Some fields would be required by the controller (IDs for example for doing modify/delete operations), but the other fields (username, year, major, etc..) the controller doesn't really need them.

Alternative approach would be to build your UserData outside of controller, and pass in the whole UserData instance into addUser.  This way you can change the data model (eg. add or remove fields) without having to touch your controller.

```
ctr.addUser(datastore, new UserData(userID, authID, userName, etc..))
```

(I believe my TaskBlocker example passes the individual fields, I apologize that was a lazy example)